.github/workflows/stackhpc-all-in-one.yml

```diff
@@ -209,32 +235,51 @@ jobs:
       # The same tag may be reused (e.g. pr-123), so ensure we have the latest image.
       - name: Pull latest Kayobe image
         run: |
-          sudo docker image pull $KAYOBE_IMAGE
+          docker image pull $KAYOBE_IMAGE
+      
+      # - name: Find and then test ~/.ssh/id_rsa to see if the file or the repo exists.
+      #   run: |
+      #     find . -type d -name "*ssh*" 2>/dev/null && \
+      #     find . -type f -name "*id_rsa*" 2>/dev/null && \
+      #     sudo ls -ltra ~/.ssh/id_rsa && \
+      #     sudo ls -ltra ~/.ssh/ && \
+      #   env:
+      #     KAYOBE_AUTOMATION_SSH_PRIVATE_KEY: ${{ steps.ssh_key.outputs.ssh_key }}
+
 
       - name: Run growroot
         run: |
-          sudo -E docker run -t --rm \
+          docker run -t --rm -u 1001 \
             -v $(pwd):/stack/kayobe-automation-env/src/kayobe-config \
+            -v ~/.ssh:/.ssh \
```

Comment:
I don't think this is required.  Same for others.

Reply:
Removed.